.github/workflows/release-grapesjs-react-latest.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish GrapesJS React latest
+on:
+  push:
+    branches: [main]
+
+jobs:
+  publish:
+    if: "contains(github.event.head_commit.message, 'Release GrapesJS React latest:')"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup project
+        uses: ./.github/actions/setup-project
+      - name: Build
+        run: yarn build:core
+      - name: Publish to npm
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
@artf we need to add this to the repo.

Reply:
I created a new one at the org level: ORG_NPM_TOKEN